src/engraving/libmscore/barline.cpp

```diff
@@ -124,7 +124,7 @@ static void undoChangeBarLineType(BarLine* bl, BarLineType barType, bool allStav
                 BarLine* sbl = toBarLine(e);
                 for (Staff* lstaff : sbl->staff()->staffList()) {
                     Score* lscore = lstaff->score();
-                    int ltrack = lstaff->idx() * VOICES;
+                    staff_idx_t ltrack = lstaff->idx() * VOICES;
```

Comment:
`track_idx_t`?

Reply:
Well, `lstaff->idx()` gives a `staff_idx_t`, it is chat the variable __name__ `ltrack` that suggests it ti be a track (and it is later used as such too. Under the cober both types are `size_t` anyway, so ti doesn't really matter, or does it?

Reply:
Well, `lstaff->idx()` gives a `staff_idx_t`, it is just the variable __name__ `ltrack` that suggests it to be a `track_idx_t` (and it is later used as such too). Under the cover both types are `size_t` anyway, so it doesn't really matter, or does it?

Reply:
So it seems a matter of coding style, use the type suitable for initialization, or the type for the later use (and cast if need be on the other occasions, not needed here though)?